themes/default/upload_image.tpl

```diff
@@ -7,134 +7,286 @@
-.ok           { padding: 0px 0px 0px 20px; font-weight:bold; color:red; background-image:url({/literal}{$THEMES_DIR}/{$settings.theme}{literal}/images/tick.png); background-repeat:no-repeat; background-position: left; }
-img.delete    { max-width:300px; max-height:150px; }
-img.uploaded  { max-width:300px; max-height:110px; /*cursor:pointer;*/ }
-img.browse    { max-width:320px; cursor:pointer; }
-.deletelink   { font-size:11px; padding-left:13px; background:url({/literal}{$THEMES_DIR}/{$settings.theme}{literal}/images/bg_sprite_3.png) no-repeat 0 -47px; }
-.small        { font-size:11px; line-height:16px; }
-code          { font-family:"courier new", courier; color:#000080; }
-a:link        { color:#0000cc; text-decoration: none; }
-a:visited     { color:#0000cc; text-decoration: none; }
-a:hover       { color:#0000ff; text-decoration: underline; }
-a:active      { color:#ff0000; text-decoration: none; }
-table         { width:100%; margin:5px 0 0 0; padding:0; }
-td            { text-align:center; }
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
+body {
+  color: #000;
+  background: #fff;
+  margin: 0;
+  padding: 0;
+  font-family: verdana, arial, sans-serif;
+  font-size: 1em;
+  font-size: 1rem;
+  line-height: 1.5;
+}
+body > * {
+  padding-block: 0;
+  padding-inline: 0.5em;
+}
+header {
+  margin: 0;
+  background: #f9f9f9;
+  border-bottom: 1px solid #bacbdf;
+  display: flex;
+  justify-content: space-between;
+  position: sticky;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+header > * {
+  margin:0;
+  padding:0;
+}
+main {
+  margin-inline: 0;
+  margin-block: 0.5em;
+}
+h1 {
+  font-size: 1em;
+  font-weight: bold;
+}
+.caution,
+.ok {
+  margin-block: 0.5em;
+  padding: 0 0 0 24px;
+  background-repeat:no-repeat;
+  background-position: 2px 3px;
+}
+.caution h2,
+.ok h2 {
+  font-size: 1em;
+  margin: 0 0 0.5em 0;
+}
+.caution {
+  color: #cc0000;
+  background-image:url({/literal}{$THEMES_DIR}/{$settings.theme}{literal}/images/caution.png);
+}
+.ok {
+  color:green;
+  background-image:url({/literal}{$THEMES_DIR}/{$settings.theme}{literal}/images/tick.png);
+}
+.deletelink {
+  padding-left:13px;
+  background:url({/literal}{$THEMES_DIR}/{$settings.theme}{literal}/images/bg_sprite_3.png) no-repeat 1px -45px;
+}
+.insert-desc,
+.deletelink,
+.small {
+  font-size: 0.82em;
+}
+code {
+  font-family:"courier new", courier, monospace;
+  color:#000080;
+}
+a {
+  color:#00c;
+  text-decoration: none;
+}
+a:focus,
+a:hover {
+  color:#00f;
+  text-decoration: underline dotted 9% #45f;
+}
+a:active {
+  color:#f00;
+  text-decoration: underline solid 7% #d00;
+}
+form > div:not(:last-child) {
+  margin-block-end: .75em;
+}
+ul {
+  list-style:none;
+  margin-block:0.5em;
+  padding: 0;
+}
+#imgtab {
+  display:flex;
+  flex-direction:column;
+  gap:0.75em;
+}
+#del-upload-form,
+#imgtab li {
+  text-align:center;
+}
+li > *:last-child {
+  align-content:center;
+}
```

Comment:
The centered text (DE: _Bitte auf das Bild klicken, um es in den Eintrag einzufügen_.) looks unusual and I would rather place it on the left. However, this is not really an issue.

```
#imgtab li {
  text-align:center;
}
```

Reply:
I tested the popup in many different page sizes from a few hundred pixels wide popup to a more or less full sized browser windows with a width of more than 1200 pixels. With some elements centered and other elements aligned at left|right one optically loses the context (IMHO). Because of this I centered more or less all elements with exception of headings and system messages.

Reply:
I meanwhile solved the *missing* formattings for the ionsert image explanations. When testing I saw the centered explanations and played with the width of the popup viewport. That's really not nice to look at. I will check how it looks with restricting the explanations width to the width of the list items but with `text-align: start;`. Even then, however, line breaks may occur.

Reply:
A note: the code `align-content: center;` does not affect the centered text, this would be `text-align: center;`. With `align-content: center` one can achieve a vertical alignment of elements, that are not displayed in a flexbox, in a grid or in a table cell.

Reply:
This got fixed with the commits 80cc239687 and b481980504.